src/Microsoft.Azure.SignalR.AspNet/OwinExtensions.cs

```diff
@@ -181,6 +181,15 @@ private static void RunAzureSignalRCore(IAppBuilder builder, string applicationN
                 throw new ArgumentException(nameof(applicationName), "Empty application name is not allowed.");
             }
 
+            if (options.UseHubNamePrefix)
```

Comment:
no need to `UseHubNamePrefix`, let's just enable it

Reply:
OK, I've removed this switch

Reply:
I've just recalled why I added this.   I was concerned that if there is someone with multiple app servers using different versions of the library (say during rolling out an update), then some will be connected to prefixed hub names and some not prefixed?

Reply:
Nice catch! Yes, that's a breaking change in deed. I will point that out in release note.